src/test/scala/magellan/OsmSuite.scala

```diff
@@ -1,160 +0,0 @@
-package magellan
-
-import org.scalatest.FunSuite
-import org.apache.spark.sql.magellan.MagellanContext
```

Comment:
@halfabrane in https://github.com/halfabrane/magellan/blob/MAGELLAN-SPARK1.6/python/magellan/context.py#L67 we are still loading `MagellanContext` This breaks the Python API.  `wclass = loader.loadClass("org.apache.spark.sql.magellan.MagellanContext")`